invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/BookCard/BookCard.js

```diff
@@ -2,29 +2,23 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Card, Image, Label } from 'semantic-ui-react';
 import { FrontSiteRoutes } from '../../../../routes/urls';
-import { CARD_IMAGE_SIZE, getCover } from '../../config';
+import { getCover } from '../../config';
 import { goTo } from '../../../../history';
+import { DocumentAuthors } from '../../Documents/DocumentsDetails/components/DocumentMetadata/components';
 
 export class BookCard extends Component {
   renderLabels = meta => {
-    return (
-      <>
-        {meta.circulation.has_items_for_loan > 0 ? (
-          <Label
-            attached="bottom left"
-            icon="book"
-            title="The book is currently avaialble"
-          />
-        ) : null}
-        {meta.eitems.total > 0 ? (
-          <Label
-            attached="bottom right"
-            icon="computer"
-            title="An electronic version of the book is avaialble"
-          />
-        ) : null}
-      </>
-    );
+    if (meta.circulation.has_items_for_loan > 0 || meta.eitems.total > 0) {
```

Comment:
nit: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator
```
return (
  <>
    {meta.circulation.has_items_for_loan.length && <Label>On shelf</Label>}
    {meta.eitems.total > 0 && <Label>E-book</Label>}
  </>
```
and you don't need all the `null` normally. :)